Makefile

```diff
@@ -305,31 +294,38 @@ create-cluster-management: $(CLUSTERCTL) ## Create a development Kubernetes clus
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		create -f examples/_out/controlplane.yaml
-	# Get KubeConfig using clusterctl.
-	$(CLUSTERCTL) \
-		alpha phases get-kubeconfig -v=3 \
-		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		--namespace=default \
-		--cluster-name=$(CLUSTER_NAME)
-	# Apply addons on the target cluster, waiting for the control-plane to become available.
-	$(CLUSTERCTL) \
-		alpha phases apply-addons -v=10 \
-		--kubeconfig=./kubeconfig \
-		-a examples/addons.yaml
 	# Create a worker node with MachineDeployment.
 	kubectl \
```

Comment:
would it make sense to move the worker node create to _after_ the CNI is applied since we don't need the machines to be created to fetch the kubeconfig?

Reply:
or maybe change the wait script to not wait for all machines

Reply:
that way we do start the create asap